src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+- name: Checking DNS resolution first
+  shell: |
+    dig ipa-ca.{{ ipa_domain }} | grep NOERROR
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 10
+
+- block:
+  - name: Run IPA advise script for server setup for smart cards
+    shell: |
+      kinit admin@{{ ipa_domain.upper() }}
+      ipa-advise config-server-for-smart-card-auth > /tmp/sc_server.sh
+      sh -x /tmp/sc_server.sh {{ sssd_ca_path }} {{ ipa_ca_path }}
+    args:
+      stdin: '{{ ipa_password }}'
+    register: ipa_advise_sc_server
+
+  - name: Workaround to disable IPA WebUI OCSP checking
+    lineinfile:
+      path: /etc/httpd/conf.d/ssl.conf
+      regexp: 'SSLOCSPEnable on'
+      line: 'SSLOCSPEnable on no_ocsp_for_cert_ok'
+      backrefs: yes
+
+  - name: Restart httpd service
+    ansible.builtin.systemd_service:
+      name: httpd
+      state: restarted
+  when: inventory_hostname == groups.ipa.0
+
+- block:
+  - name: Get IPA advise script for client setup for smart cards from IPA server
+    shell: |
```

Comment:
Would command: work here?

Reply:
Yes, I think so.   Will try with that to be sure.